src/OrchardCore.Modules/OrchardCore.ReverseProxy/Services/IReverseProxyService.cs

```diff
@@ -0,0 +1,10 @@
+using System.Threading.Tasks;
+using OrchardCore.ReverseProxy.Settings;
+
+namespace OrchardCore.ReverseProxy.Services
```

Comment:
Theorically, as long as the interface is kept in the module is kept in the module, you don't need an interface.